guides/NEW_APP.md

```diff
@@ -0,0 +1,50 @@
+Creating a new app
+==================
+
+The following steps will guide you through add adding a new app to our system that is built by the CI system and deployed to the live environment.
+
+Prerequisites
+-------------
+
+In order to follow this guide you will need:
+
+* A github account with access to the ONSdigital organisation
+
+* A login for the concourse CI environment
+
+* [dp-cli](https://github.com/ONSdigital/dp-cli)
+
+* Git:
+
+  ```
+  brew install git
+  ```
+
+* Terraform >= 0.12:
+
+  ```
+  brew install terraform
+  ```
+
+* Fly cli (can be downloaded from the links in the corner of the concourse UI)
+
+Getting started
+---------------
+
+These steps will create a new app that is built by CI and deployed to the develop and production environments.  If any of the following github links 404 it is likely that you need to be added to the ONSdigital organisation.
+
+1. Determine what language you want to use and technology your app will likely use
+
+2. Pick a local dev port for your app and add it to the [local ports list](PORTS.md)
+
+3. Use the [`dp-cli`](https://github.com/ONSdigital/dp-cli) to generate the new repo and populate it with the standard boilerplate files:
+
+   ```
+   dp-utils generate-project
```

Comment:
So yes this will generate a project but won't add it to Github necessarily, there is a flag that needs to be provided. I would suggest linking to the readme.md here but that is still in a PR - this will be where it lives mind, but I can come back and PR and edit this later.
https://github.com/ONSdigital/dp-cli/blob/fbc8fa72b1c4052541c2943efde2af56655c3237/project-generation/COMPLETE_PROJECT_SETUP.md

Reply:
Adding a link to where that README will be. Does mean it is broken in the short term just until your PR is merged.